### PR TITLE
Document modification script

### PR DESCRIPTION
**Pull request by RichardJFoster**
Just a couple of paragraphs about the chocolateyBeforeModify.ps1 script and when it will be triggered.

### CODE REVIEW

**Comment by RichardJFoster** [comment]
Er... not sure why it's showing 14 changes. There should only be one (567ae096ad) feel free to just cherry pick that commit.

File: CreatePackages.md [hunk 1]
```diff
@@ -97,6 +97,19 @@ $silentArgs = '/S'
 Install-ChocolateyPackage $packageName $fileType $silentArgs $url
 ```
 
+## During which scenarios will my custom scripts be triggered?
+The table below shows which scripts are available, and which command(s) will cause them to be run.
+
+Script Name                                    | Install | Upgrade | Uninstall
+-----------------------------------------------|---------|---------|----------
+chocolateyInstall.ps1                          | Yes     | Yes     |
+chocolateyBeforeModify.ps1                     |         | Yes     | Yes
```
Comment (ferventcoder) [inline CreatePackages.md]: Can you move this to before chocolateyInstall.ps1? That way there is no confusion on the order.

File: CreatePackages.md [hunk 2]
```diff
@@ -158,7 +171,11 @@ If you allow customizing the installation path, then append instructions on how
 
 ## Upgrading
 
-There is no automation script for just upgrade. Instead, your [[chocolateyInstall.ps1|ChocolateyInstallPS1]] script should support installing/upgrading on top of any previous versions of your package.
+Prior to choco version 0.9.10, there is no dedicated automation script for upgrade scenarios. Instead, your [[chocolateyInstall.ps1|ChocolateyInstallPS1]] script should support installing/upgrading on top of any previous versions of your package.
+
+More recent versions of choco give you the option of supplying a `chocolateyBeforeModify.ps1` script.
```
Comment (ferventcoder) [inline CreatePackages.md]: "More recent versions of choco (0.9.10+)"

**Comment by RichardJFoster** [comment]
Second attempt now available (without unnecessary whitespace changes and content updated as suggested).

**Pull request merged**